src/ansys/grantami/recordlists/_connection.py

```diff
@@ -339,14 +339,13 @@ def update_list(
         if name is None:
             raise ValueError(f"If provided, argument 'name' cannot be None.")
 
-        body = []
+        body = models.GrantaServerApiListsDtoUpdateRecordListProperties()
         if name != _ArgNotProvided:
-            body.append(self._create_patch_operation(name, "name"))
+            body.name = name
         if description != _ArgNotProvided:
-            body.append(self._create_patch_operation(description, "description"))
+            body.description = description if description else ""
```

Comment:
I’m not sure about this. I need to investigate more about how to clear a value here, because I don’t think our current code distinguishes between None and unset on a model.